Sources/StreamChatSwiftUI/ChatChannel/Composer/SlowModeView.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+
+/// View displaying slow mode countdown.
+public struct SlowModeView: View {
+    
+    @Injected(\.colors) private var colors
+    @Injected(\.fonts) private var fonts
+    
+    private let size: CGFloat = 32
+    
+    var cooldownDuration: Int
+    
+    public var body: some View {
+        Text("\(cooldownDuration)")
+            .padding(.all, 8)
+            .font(fonts.bodyBold)
+            .frame(width: cooldownDuration < 10 ? size : nil, height: size)
```

Comment:
I like this more but it's very subjective. :) 

```suggestion
            .padding(.horizontal, 8)
            .font(fonts.bodyBold)
            .frame(width: cooldownDuration < 10 ? size : nil, height: size)

```